types/chrome/test/index.ts

```diff
@@ -2290,15 +2290,27 @@ function testUserScripts() {
     chrome.userScripts.update(scripts, () => void 0); // $ExpectType void
 }
 
-function testPlatformKeys() {
+// https://developer.chrome.com/docs/extensions/reference/api/enterprise/platformKeys
+function testEnterPrisePlatformKeys() {
+    chrome.enterprise.platformKeys.Scope;
+    chrome.enterprise.platformKeys.Algorithm;
```

Comment:
Being nitpicky here, any reason that there's no ExpectType here? Also, any reason to not test that the ${Algorithm} works correctly?

Reply:
I strengthened the tests💪